tests/tests.bats

```diff
@@ -187,3 +187,27 @@ DOCKER_PLUGIN_DEFAULT_ARG="/usr/sbin/sshd -D -p 22"
   run docker run --entrypoint sh --rm "${SUT_IMAGE}" -c 'locale charmap'
   assert_equal "${output}" "UTF-8"
 }
+
+
+@test "[${SUT_IMAGE}] image has ssh client, less and patch installed and in the PATH" {
+  local test_container_name=${AGENT_CONTAINER}-bash-java
+  clean_test_container "${test_container_name}"
+  docker run --name="${test_container_name}" --name="${test_container_name}" "${docker_run_opts[@]}" "${PUBLIC_SSH_KEY}"
+
+  run docker exec "${test_container_name}" sh -c "command -v ssh"
+  assert_success
+  run docker exec "${test_container_name}" ssh -V
+  assert_success
+
+  run docker exec "${test_container_name}" sh -c "command -v less"
+  assert_success
+  run docker exec "${test_container_name}" less -V
```

Comment:
This test fails for Alpine image ass the flag `-V`does not exist for the busybox version of `less`.

It is a good thing! It shows that we need to install the GNU `less` with `apk add --no-cache less` for the Alpine image (see my other comment about splitting the PR).

Reply:
I'm glad you spotted that, thanks. 😊